### PR TITLE
Tweak import preview page UI

### PR DESCRIPTION
**Pull request by rowanseymour**
So when adding to a group isn't enabled, the group controls are disabled

Captura de Pantalla 2021-05-06 a la(s) 11 33 10

Captura de Pantalla 2021-05-06 a la(s) 11 33 17

The select in this case doesn't appear different, I think due to https://github.com/nyaruka/temba-components/issues/74

### CODE REVIEW

File: templates/contacts/contactimport_preview.haml [hunk 1]
```diff
@@ -92,13 +92,32 @@
...
       });
 
       updateGroupMode("{{ form.group_mode.value }}");
+
+      var addToGroup = document.querySelector("temba-checkbox[name='add_to_group']");
+      addToGroup.addEventListener('change', function(event) {
+        updateAddToGroup(event.target.checked)
+      });
+
+      updateAddToGroup({{ form.add_to_group.value|lower }});
```
Comment (ericnewcomer) [inline templates/contacts/contactimport_preview.haml:120]: My preference is to use an attribute to bind these events instead, `onchange="..."`

Comment (rowanseymour) [reply templates/contacts/contactimport_preview.haml:120]: You mean setting that thru `SelectWidget(attrs={})` in Django?

Comment (ericnewcomer) [reply templates/contacts/contactimport_preview.haml:120]: Hmmm.. ya, was more thinking vanilla html. But ya, I suppose that should be no different.

Comment (rowanseymour) [reply templates/contacts/contactimport_preview.haml:120]: I don't love the idea of putting Javascript in Django forms.. you want me to replace the `render_field` tag with the generated HTML?

**Pull request merged**